frontend/src/commons/AddDropdownMenu.js

```diff
@@ -105,14 +105,14 @@ export default function AddDropdownMenu() {
 }
 
 const DropdownItemStyled = styled.div`
-  height: 50px;
+  height: 20px;
```

Comment:
Maybe let the content control the height via it's font size. When you set padding in relation to root font size, you might create problems when the user changes the default font size of the browser.